htdocs/core/actions_extrafields.inc.php

```diff
@@ -367,7 +367,7 @@
 					$computedvalue,
 					(GETPOST('entitycurrentorall', 'alpha') ? 0 : ''),
 					GETPOST('langfile'),
-					GETPOST('enabled', 'alpha'),
+					GETPOST('enabled', 'restricthtml'),
```

Comment:
Seems not enough protected. Can you try with 'nohtml' instead of 'restricthtml' ?

Reply:
I made test, it's ok